.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "5"
+before_script:
+  - npm install grunt-cli -g
+env:
+  - CXX=g++-4.8  
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
+      - g++-4.8
```

Comment:
i don't think the g++-4.8 stuff is necessary. Did you get that from DNSChain's file? I think that's included there before some dependency (that we're not using) needs it